fulp_modules/antag_tips/html/changeling.html

```diff
@@ -0,0 +1,15 @@
+<html>
+	<center>
+	<h1>You are a Changeling!</h1>
+	<img src="changeling.gif"/>
+	<p>You are a Changeling, a shapeshifting alien assuming the form of a crewmember on Space Station 13.</p>
+	<p>The goal of many changelings is to acquire 5-7 DNA strains. To do this, it must take ANY human, living or dead, and acquire their DNA.</p>
```

Comment:
I don't see why we should bring up that its a possible objective, just say its a way to steal DNA